core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.spec.ts

```diff
@@ -355,6 +374,31 @@ describe('ContainerListComponent', () => {
                 '123Archived'
             ]);
         });
+
+        it('should hide action of file or system container', () => {
+            const systemContainerActions = fixture.debugElement
+                .query(By.css('[data-testrowid="SYSTEM_CONTAINER"]'))
```

Comment:
```suggestion
                .query(By.css('[data-testId="SYSTEM_CONTAINER"]'))
```

Reply:
It is added in `dot-listing-data-table` reusable component if we change here we need to change all other test cases where it is used.